### DISCUSSION TITLE
请问大家，游客权限只勾选了无密码范围，但是游客还是有下载按钮和复制下载链接的按钮

### DISCUSSION THREAD

**Discussion by luciferrrrrrrr** [AlistGo/alist#6638] [Q&A]
<img width="821" alt="image" src="https://github.com/alist-org/alist/assets/64362071/648c0fe9-6c6a-4560-8791-5816023e144f">
<img width="1150" alt="image" src="https://github.com/alist-org/alist/assets/64362071/0f457337-cb85-4a4a-8a60-cc17b779e4de">

如图，只是想把网盘作为目录展示，不想像游客提供下载选项。请问各位大佬应该如何配置实现呢？

**Comment by pongfcnkl** [comment]
无密码不打勾才是需要密码，而且这个和能不能下载无关，不打勾就需要游客输入密码了
至于不想让别人下载，可以去代理url里随便填点东西就行了，变相实现不可下载

**Comment by zyk121381** [comment]
如果不希望用户下载，同时文档图片什么的也不对用户开放（只开放文件列表），那可以再代理URL随意填写。如果希望文档图片什么的游客可以正常预览，仅仅只是限制下载功能的话，可以通过在alist后台全局设置的自定义头部自己写css文件，把下拉下载的按钮隐藏。如果本身没有预览窗格，点开文件即下载，那么可以写css将这两个按钮隐藏。

提示：用户长按（移动端）/右键（PC）文件列表也可调出复制链接和下载按钮。如果需要防止用户下载，记得把这两个也写到css里面
